src/test/java/mate/academy/dao/UserDaoTest.java

```diff
@@ -0,0 +1,98 @@
+package mate.academy.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import mate.academy.dao.impl.UserDaoImpl;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoTest extends AbstractTest {
+    private static final Long ID = 1L;
+    private static final Long NOT_EXISTING_ID = 100L;
+    private static final String EMAIL = "bob@mate.academy";
+    private static final String NOT_EXISTING_EMAIL = "alice@mate.academy";
+    private static final String PASSWORD = "12345678";
+    private static User bob;
+    private UserDao userDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        bob = new User();
+        bob.setEmail(EMAIL);
+        bob.setPassword(PASSWORD);
+    }
+
+    @BeforeEach
+    void setUp() {
+        userDao = new UserDaoImpl(getSessionFactory());
```

Comment:
move dao to beforeAll everywhere

Reply:
Can you help me with th it? Because I can't do it, creating dao object require calling getSessionFactory() and this method is not static, so I can not move it to beforeAll. And I don't know any other ways to create dao object once per class, not before every test